util/importers/LDC97S62.py

```diff
@@ -175,36 +180,32 @@ def _maybe_convert_wav(data_dir, original_data, converted_data):
 
 def _parse_transcriptions(trans_file):
     segments = []
-    with open(trans_file, "r") as fin:
+    with codecs.open(trans_file, "r", "utf-8") as fin:
         for line in fin:
             if line.startswith("#")  or len(line) <= 1:
                 continue
 
-            filename_time_beg = 0;
-            filename_time_end = line.find(" ", filename_time_beg)
+            tokens = line.split()
+            start_time = float(tokens[1])
+            stop_time = float(tokens[2])
+            transcript = validate_label(" ".join(tokens[3:]))
 
-            start_time_beg = filename_time_end + 1
-            start_time_end = line.find(" ", start_time_beg)
-
-            stop_time_beg = start_time_end + 1
-            stop_time_end = line.find(" ", stop_time_beg)
-
-            transcript_beg = stop_time_end + 1
-            transcript_end = len(line)
-
-            if validate_label(line[transcript_beg:transcript_end].strip()) == None:
+            if transcript == None:
                 continue
 
+            transcript = unicodedata.normalize("NFKD", transcript)  \
```

Comment:
Could you include the original comment on why this encode/decode dance is required?